src/main/java/utils/NumberFormatter.java

```diff
@@ -0,0 +1,34 @@
+package utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class NumberFormatter {
+    private final static String COMMA = ",";
+    public final static List<Integer> formedNumbers = new ArrayList<>();
+
+    public static List<Integer> formNumbers(String inputNumbers) {
+        checkNull(inputNumbers);
+
+        try {
+            formedNumbers.clear();
+            formedNumbers.addAll(
+                    Arrays.stream(inputNumbers.split(COMMA)) // "1,2,3,4,5" → ["1", "2", "3", "4", "5"]
+                            .map(Integer::parseInt) // ["1", "2", "3", "4", "5"] → [1, 2, 3, 4, 5]
+                            .sorted() // 오름차순으로 정렬
+                            .toList() // [1, 2, 3, 4, 5]
+            );
+        } catch (NumberFormatException e){
+            throw new IllegalArgumentException("입력 값에 숫자가 아닌 값이 포함되어 있습니다.");
+        }
+
+        return formedNumbers;
+    }
+
+    private static void checkNull(String inputNumbers){
+        if (inputNumbers == null || inputNumbers.trim().isEmpty()) {
```

Comment:
```suggestion
        if (inputNumbers == null || inputNumbers.isBlank()) {
```
P2) Java 11에 `isBlank()`가 추가되어서 `trim().isEmpty()`를 대체할 수 있어요 ([참고글](https://velog.io/@jennyfromdeblock/isBlank-isEmpty-isNull-차이점))